ale_linters/ruby/brakeman.vim

```diff
@@ -0,0 +1,83 @@
+" Author: Eddie Lebow https://github.com/elebow
+" Description: Brakeman, a static analyzer for Rails security
+
+function! ale_linters#ruby#brakeman#Handle(buffer, lines) abort
+    let l:result = json_decode(join(a:lines, ''))
+
+    let l:output = []
+
+    for l:warning in l:result.warnings
+        if !s:MatchFileWithoutRailsRoot(l:warning.file, a:buffer)
```

Comment:
This function exists now. Try using `ale#path#IsBufferPath(a:buffer, l:warning.file)`. You can pass a relative path or an absolute path, and `1` will be returned if the filenames represent the same file. (It should work like that anyway.)